src/app/components/filter/filter-options.component.ts

```diff
@@ -0,0 +1,23 @@
+import { OverlayModule } from '@angular/cdk/overlay';
+import { CommonModule } from '@angular/common';
+import { Component } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+import { FilterOptionModel, MuziehComponentsModule, OptionsFilterComponent } from 'muzieh-ngcomponents';
+import { FilterComponent } from "../../../../projects/components/src/lib/filter/filter.component";
```

Comment:
Can we import it through line 5 instead?

Reply:
done